mkdocs.yml

```diff
@@ -245,6 +245,7 @@ nav:
         - data/sdks/unreal/index.md
       - GO:
         - data/sdks/go/index.md
+        - Ampli Codegen: data/sdks/go/ampli.md
```

Comment:
I think we should mark these as alpha e.g. `Go (Alpha)` and `Ampli Codegen (Alpha)`